RFS/src/testFixtures/java/org/opensearch/migrations/bulkload/framework/SearchClusterContainer.java

```diff
@@ -114,7 +114,9 @@ public SearchClusterContainer(final ContainerVersion version) {
     public void copySnapshotData(final String directory) {
         try {
             // Execute command to list all files in the directory
-            final var result = this.execInContainer("sh", "-c", "find " + CLUSTER_SNAPSHOT_DIR + " -type f");
+            // `find` is not available on some versions of the containers, in which case it falls back to a ls/grep loop.
+            final var result = this.execInContainer("sh", "-c", "find " + CLUSTER_SNAPSHOT_DIR + " -type f" + " || " +
```

Comment:
Can you open an issue in https://github.com/opensearch-project/opensearch-build this seems like an unintentional regression

Reply:
I assume it's because they're trying to minimize the container size and therefore is an intentionally exclusion. So far as I know, it's all 2.x images, not just 2.19.1. Do you have additional reason to believe it's unintentional?

Reply:
I mean that seems like a reasonable reason for them to push back, but `find` seems pretty standard IMO.  I say unintentional because they might not realize the impact.  If grep went away we'd also have trouble even if it was for the same reason.